website/client/js/controllers/rootCtrl.js

```diff
@@ -6,6 +6,9 @@
 habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$http', '$state', '$stateParams', 'Notification', 'Groups', 'Shared', 'Content', '$modal', '$timeout', 'ApiUrl', 'Payments','$sce','$window','Analytics','TAVERN_ID',
   function($scope, $rootScope, $location, User, $http, $state, $stateParams, Notification, Groups, Shared, Content, $modal, $timeout, ApiUrl, Payments, $sce, $window, Analytics, TAVERN_ID) {
     var user = User.user;
+    var IGNORE_SCROLL_PAGES = {
+        'options.social.challenges.detail': true
```

Comment:
Thanks! But you know I couldn't have done it without your help, so well done to you :smile: